examples/testsuites/helloworld.py

```diff
@@ -1,9 +1,9 @@
-from lisa import TestCaseMetadata, TestSuite, TestSuiteMetadata
+from lisa import LisaTestCase, LisaTestCaseMetadata, LisaTestMetadata
 from lisa.operating_system import Linux
 from lisa.tools import Echo, Uname
 
 
-@TestSuiteMetadata(
+@LisaTestCaseMetadata(
```

Comment:
Keep class -> suite, and method -> case. See my explanation in overall comments, we have different concept on them than Python ut. We're not write UT, so the concept of suite and case are heavier than Python UT.